src/Base/UnitsSchemaImperial1.cpp

```diff
@@ -30,12 +30,8 @@
 #endif
 
 #include <QString>
-#include "Console.h"
-#include "Exception.h"
-#include "UnitsApi.h"
+
 #include "UnitsSchemaImperial1.h"
-#include <cmath>

```

Comment:
This fails to compile on Linux, it needs the math functions `round` and `floor`.

Reply:
Thanks. I re-added it now